scripts/cli/tests/4_permission_management.js

```diff
@@ -11,26 +12,86 @@ async function main() {
 
   const api = await reqImports.createApi();
 
+  const ticker = await reqImports.generateRandomTicker(api);
+
+  const portfolioName = await reqImports.generateRandomTicker(api);
+
   let primary_dev_seed = await reqImports.generateRandomKey(api);
   
   let secondary_dev_seed = await reqImports.generateRandomKey(api);
-  
 
   const testEntities = await reqImports.initMain(api);
 
-  let primary_keys = await reqImports.generateKeys(api, 2, primary_dev_seed );
+  const alice = testEntities[0];
 
-  let secondary_keys = await reqImports.generateKeys(api, 2, secondary_dev_seed );
+  let primary_keys = await reqImports.generateKeys(api, 1, primary_dev_seed );
 
-  let issuer_dids = await reqImports.createIdentities(api, primary_keys, testEntities[0]);
+  let secondary_keys = await reqImports.generateKeys(api, 1, secondary_dev_seed );
+  
+  let issuer_dids = await reqImports.createIdentities(api, primary_keys, alice);
 
-  await reqImports.distributePolyBatch( api, primary_keys, reqImports.transfer_amount, testEntities[0] );
+  let extrinsics = [];
+  let portfolios = [];
+  let assets = [];
+  let documents = [];
+  
+  await reqImports.distributePolyBatch( api, [primary_keys[0]], reqImports.transfer_amount, alice );
```

Comment:
There is too much empty space in the PR, both horizontal and vertical:
* extra space afer `(` and before `)`
* instructions could be grouped in semantic blocks within which there is no need for extra empty lines between instructions.

Reply:
understood, would make the change. Thanks for the feedback.